src/pkg/services/m365/api/groups.go

```diff
@@ -113,10 +114,6 @@ func (c Groups) GetTeamByID(
 
 	t, err := c.Stable.Client().Teams().ByTeamId(identifier).Get(ctx, nil)
 	if err != nil {
-		if graph.IsErrResourceLocked(err) {
-			return nil, graph.Stack(ctx, clues.Stack(graph.ErrResourceLocked, err))
-		}
-
 		return nil, graph.Wrap(ctx, err, "finding team by ID")
```

Comment:
nit: the errors in most/all of these calls with changes can just be clues calls now right? The lower layers should now add the error details when it calls `stackWithCoreErr` no?

```suggestion
		return nil, clues.Wrap(err, "finding team by ID")
```